src/Avalonia.Controls/ScrollViewer.cs

```diff
@@ -713,5 +714,44 @@ private void RaiseScrollChanged()
                 _oldViewport = Viewport;
             }
         }
+
+        /// <summary>
+        /// Defines the PointerWheelScrollMode property.
+        /// </summary>
+        public static readonly DirectProperty<ScrollViewer, PointerWheelScrollMode> PointerWheelScrollModeProperty =
+            AvaloniaProperty.RegisterDirect<ScrollViewer, PointerWheelScrollMode>(
+                nameof(PointerWheelScrollMode),
+                o => o.PointerWheelScrollMode,
+                (o, v) => o.PointerWheelScrollMode = v);
+
+        /// <summary>
+        /// Gets or sets the pointer wheel scroll mode scrollbar value.
+        /// </summary>
+        public PointerWheelScrollMode PointerWheelScrollMode
+        {
+            get { return _pointerWheelScrollMode; }
+            set
+            {
+                if (_pointerWheelScrollMode != value)
+                {
+                    var old = _pointerWheelScrollMode;
+                    _pointerWheelScrollMode = value;
+                    RaisePropertyChanged(PointerWheelScrollModeProperty, old, value);
+                }
+            }
+        }
+
+        protected override void OnPointerWheelChanged(PointerWheelEventArgs e)
```

Comment:
The event is handled in https://github.com/AvaloniaUI/Avalonia/blob/fec8341176acf383233e218422c613ad9fc3adbf/src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs#L396

Why do we need a separate handler here?

Reply:
It is my mistake. I Close PR.